Task2/Task2.cs

```diff
@@ -12,9 +12,163 @@ namespace cg_lab3.Task2
 {
     public partial class Task2 : Form
     {
+        Bitmap ImageBitmap;
```

Comment:
неверное наименование переменной
```suggestion
        Bitmap imageBitmap;
```